.github/workflows/docker.yml

```diff
@@ -83,6 +81,8 @@ jobs:
         with:
           context: .
           push: true
+          secrets: |
```

Comment:
Also needed to adjust the dockerfile & docker-action. 
I'm not sure here if it works as expected we probably have to test it first.